libs/wire-api/src/Wire/API/Error/Brig.hs

```diff
@@ -170,6 +171,8 @@ type instance MapError 'InvalidTransition = 'StaticError 403 "bad-conn-update" "
 
 type instance MapError 'NoIdentity = 'StaticError 403 "no-identity" "The user has no verified email"
 
+type instance MapError 'NoUser = 'StaticError 403 "no-user" "The user does not exist"
+
```

Comment:
What was the error before the refactoring?

Reply:
"TODO: searcher doesn't exist" :)

Reply:
I was asking about before this PR started. This PR is supposed to be a refactoring, so ideally we wouldn't be inventing new user visible errors. So it'd be great to know what was the previous behaviour.

Reply:
It seems like we've introduced this error, but for no good reason. I'm refactoring the code so we don't need to throw the error.

Reply:
This type instance is gone now and we don't throw a new error anymore compared to the old code.